### PR TITLE
Prepare for rpc connection

### PR DESCRIPTION
**Pull request by yhmo**
Signed-off-by: yhmo <yihua.mo@zilliz.com>

#16

### CODE REVIEW

**Comment by ArkaprabhaChakraborty** [comment]
I think this is some file pathway error in the CMakeLists.txt which causes the build to fail 
```
Make Error at src/CMakeLists.txt:36 (add_library):
  Cannot find source file:

    /home/runner/work/milvus-sdk-cpp/milvus-sdk-cpp/src/proto-gen/milvus.grpc.pb.cc
```

File: src/proto-gen/proto-gen.cmake [hunk 1]
```diff
@@ -1,22 +1,36 @@
...
+        COMMAND ${GRPC_BINARY_DIR}/protobuf/protoc${CMAKE_EXECUTABLE_SUFFIX}
                 --cpp_out ${CMAKE_CURRENT_SOURCE_DIR}/proto-gen -I${CMAKE_CURRENT_SOURCE_DIR}/proto
                 ${CMAKE_CURRENT_SOURCE_DIR}/proto/${name}.proto
     )
-endfunction(add_proto_source name)
+endfunction(add_definition_proto name)
+
+function(add_service_proto name)
+    add_custom_command(
+        OUTPUT ${CMAKE_CURRENT_SOURCE_DIR}/proto-gen/${name}.pb.cc
```
Comment (matrixji) [inline src/proto-gen/proto-gen.cmake:14]: I guess neet change ${name}.pb.cc => ${name}.grpc.pb.cc

**Review by matrixji — CHANGES_REQUESTED** [review]
Generate proto to pc.cc need some fix, other changes look good to me

**Comment by yhmo** [comment]
Ok, I have fixed the cmake error. Still has lint error.

**Pull request merged**